rbi/core/thread_group.rbi

```diff
@@ -1,4 +1,17 @@
 # typed: __STDLIB_INTERNAL
 
 class ThreadGroup < Object
+  Default = T.let(nil, T.untyped)
```

Comment:
isn't it a `ThreadGroup`?

Reply:
Yup, it looks like you're correct.

This should be `Default = T.let(T.unsafe(nil), ThreadGroup)`, right?